config/sources/families/rockchip-rk3588-indiedroid.conf

```diff
@@ -0,0 +1,44 @@
+source "${BASH_SOURCE%/*}/include/rockchip64_common.inc"
+
+BOOTSOURCE='https://github.com/radxa/u-boot.git'
+BOOTBRANCH='branch:stable-5.10-rock5'
+BOOTPATCHDIR="legacy"
+OVERLAY_PREFIX='rockchip-rk3588'
+
+case $BRANCH in
+
+	legacy)
+		BOOTSCRIPT='boot-rk3588-legacy.cmd:boot.cmd'
+		UBOOT_COMPILER="aarch64-linux-gnu-"
+		UBOOT_USE_GCC='< 8.0'
+		BOOTDIR='u-boot-rockchip64'
+		KERNELDIR='linux-rockchip64'
+		KERNELSOURCE='https://github.com/stvhay/kernel'
```

Comment:
I think device tree can be added into https://github.com/armbian/linux-rockchip instead of adding new family and kernel source

Reply:
This is true, but some kernel patches from my tree will be needed to add support for the onboard 8821CS wifi and bluetooth driver.

Reply:
This commit updates the drivers to a more recent version of the driver: 

https://github.com/stvhay/kernel/commit/f83ff6c84ab8082a67ddb02f06bb1cb0fa652175

This commit enables them in the kernel, and has a DTS update that can probably be separated and updated:

https://github.com/stvhay/kernel/commit/f83ff6c84ab8082a67ddb02f06bb1cb0fa652175

This commit removes what was for me use of the kernel Rijndael (AES) function which for me was causing kernel panics. Maybe someone with more kernel knowledge can figure out something better

https://github.com/stvhay/kernel/commit/fe45c1db9671a29855ed323d6c5d44f130b49d83